app/components/mandatarissen/update-state.js

```diff
@@ -143,6 +151,7 @@ export default class MandatarissenUpdateState extends Component {
       !this.date ||
       this.inValidReplacement ||
       !this.hasChanges ||
+      (this.isFractieSelectorRequired && !this.selectedFractie) ||
```

Comment:
Do we need this? Isn't this why we set the allowClear based on whether the fractie is required or not?

Reply:
This is definitly needed as otherwise you can change the fractie in wijzig and you wont not able to save it